marine/marine.py

```diff
@@ -30,6 +30,8 @@ def __init__(self, lib_path: str, epan_auto_reset_count: Optional[int] = None):
         self._marine.marine_free.argtypes = [MARINE_RESULT_POINTER]
         return_code = self._marine.init_marine()
         if return_code < 0:
+            if return_code == c_int.in_dll(self._marine, 'MARINE_ALREADY_INITIALIZED_ERROR_CODE').value:
```

Comment:
I really don't like this magic string.. :(

Reply:
Suggestions?

Reply:
I like the const that was defined. I think it's better than a long magic string.

Another option might be to expose the error code as the return value of a function (`get_marine_already_initialized_err_code()`) and not a const but that's weird

Reply:
Yeah that a bit weird, @yehonatanz @shaharyarn what do you guys think?

Reply:
@tomlegkov name of a function is no less magic than name of a const, it bothers you just because the latter is in quotes.
An `AttributeError` will be raised if this const symbol won't exist anyway and neither way is detected by static analysis.

Reply:
Ok, good point.